app/views/videos/_fields.html.erb

```diff
@@ -1,36 +0,0 @@
-<%= f.inputs do %>
```

Comment:
What impact does removing this file have? Can we still see the details for a video edition on the show page?